src/wallet/walletdb.h

```diff
@@ -48,7 +48,8 @@ enum class DBErrors
     NONCRITICAL_ERROR,
     TOO_NEW,
     LOAD_FAIL,
-    NEED_REWRITE
+    NEED_REWRITE,
+    RESCAN_REQUIRED
```

Comment:
In commit "Corrupt wallet tx shouldn't trigger rescan of all wallets" (f963b0fa8cdd5223feb828c5faf6c57bc4107c8a)

Minor: Might call this `NEED_RESCAN` to be consistent with `NEED_REWRITE`

Reply:
I think this is a good suggestion to be more consistent. Maybe in a follow-up PR.